external/upstream/qcengine/CMakeLists.txt

```diff
@@ -21,7 +21,8 @@ else()
     ExternalProject_Add(qcengine_external
         DEPENDS qcelemental_external
         BUILD_ALWAYS 1
-        URL https://github.com/MolSSI/QCEngine/archive/v0.6.4.tar.gz
+        #URL https://github.com/MolSSI/QCEngine/archive/v0.7.0.tar.gz
+        URL https://github.com/MolSSI/QCEngine/archive/master.tar.gz
```

Comment:
Just out of interest, is GH cool with us using it as a CDN in this way?  I remember them being quite strictly against that (back when it was still a startup), but I can imagine them relaxing that as they got bigger.

Reply:
I never heard of GH being averse. Are you thinking an issue between GIT_TAG and URL? lgtm only accepts he latter.